packages/@uppy/companion/test/__tests__/http-agent.js

```diff
@@ -1,37 +1,7 @@
 const nock = require('nock')
-const { getRedirectEvaluator, FORBIDDEN_IP_ADDRESS, FORBIDDEN_RESOLVED_IP_ADDRESS } = require('../../src/server/helpers/request')
+const { FORBIDDEN_IP_ADDRESS } = require('../../src/server/helpers/request')
 const { getProtectedGot } = require('../../src/server/helpers/request')
 
-describe('test getRedirectEvaluator', () => {
-  const httpURL = 'http://uppy.io'
-  const httpsURL = 'https://uppy.io'
-  const httpRedirectResp = {
-    headers: {
-      location: 'http://transloadit.com',
-    },
-  }
-
-  const httpsRedirectResp = {
-    headers: {
-      location: 'https://transloadit.com',
-    },
-  }
-
-  test('when original URL has "https:" as protocol', (done) => {
-    const shouldRedirectHttps = getRedirectEvaluator(httpsURL, true)
-    expect(shouldRedirectHttps(httpsRedirectResp)).toEqual(true)
-    expect(shouldRedirectHttps(httpRedirectResp)).toEqual(false)
-    done()
-  })
-
-  test('when original URL has "http:" as protocol', (done) => {
-    const shouldRedirectHttp = getRedirectEvaluator(httpURL, true)
-    expect(shouldRedirectHttp(httpRedirectResp)).toEqual(true)
-    expect(shouldRedirectHttp(httpsRedirectResp)).toEqual(false)
-    done()
-  })
-})
-
```

Comment:
Why do we remove those tests?

Reply:
Tests are removed because the old library we used didn't support redirects to a different protocol. Now we use `got` which handles this internally.